rakelib/tests.rake

```diff
@@ -53,10 +63,15 @@ begin
       # Run RSpec tests that do not require WinRM
       desc ''
       RSpec::Core::RakeTask.new(:fast) do |t|
-        t.rspec_opts = '--tag ~winrm --tag ~windows_agents --tag ~puppetserver --tag ~puppetdb ' \
+        t.rspec_opts = '--tag ~winrm --tag ~lxd --tag ~windows_agents --tag ~puppetserver --tag ~puppetdb ' \
                        '--tag ~omi --tag ~windows --tag ~kerberos --tag ~expensive'
       end
 
+      desc 'Run tests that require an LXD virtual machine and the lxc cli'
+      RSpec::Core::RakeTask.new(:lxd) do |t|
+        t.rspec_opts = '--tag lxd'
+      end
```

Comment:
```suggestion
      desc ''
      RSpec::Core::RakeTask.new(:lxd) do |t|
        t.rspec_opts = '--tag lxd'
      end
```

We don't want the CI tasks to be visible. I'm not sure there's really a need for this separate task, since the workflow can just invoke `tests:lxd`. If we do keep this, maybe have it invoke that task?